policy/modules/roles/unconfineduser.te

```diff
@@ -158,6 +158,10 @@ optional_policy(`
         kpatch_run(unconfined_t,unconfined_r)
     ')
 
+	optional_policy(`
+		kerberos_filetrans_named_content(unconfined_t)
+	')
+
```

Comment:
You can now drop this part, since `unconfined_t` belongs to `named_filetrans_domain`, and there already is `kerberos_filetrans_named_content(named_filetrans_domain)` in policy/modules/kernel/domain.te.